test/lib/formatter_test.exs

```diff
@@ -1,15 +1,14 @@
 defmodule XcribeFormatterTest do
   use ExUnit.Case, async: false
-  use Xcribe.SwaggerExamples
-  use Xcribe.RequestsExamples
-  use Xcribe.ApiBlueprintExamples
 
   import ExUnit.CaptureIO
 
   alias Xcribe.{Formatter, Recorder, Request, Request.Error}
 
   alias Xcribe.Support.RequestsGenerator
 
+  @sample_swagger_output File.read!("test/support/swagger_example.json")
+  @sample_apib_output File.read!("test/support/api_blueprint_example.apib")
```

Comment:
Maybe this is a little overkill, but what about extracting these to a test helper module?

Reply:
It was into a module, but as we have a lot of "factory modules" it was becoming messy. I suppose I wont need read this files in other tests cases, if it is correctly I think it's a good idea keep it as a module attribute... WDYT?

Reply:
Well, as long as we don't repeat these attributes in other test files, I don't see a problem.